compiler/stable_mir/src/crate_def.rs

```diff
@@ -0,0 +1,66 @@
+//! Module that define a common trait for things that represent a crate definition.
+
+use crate::ty::Span;
+use crate::{with, Crate, Symbol};
+
+/// A unique identification number for each item accessible for the current compilation unit.
+#[derive(Clone, Copy, PartialEq, Eq, Hash)]
+pub struct DefId(pub(crate) usize);
+
+/// A trait for retrieving information about a crate definition.
+///
+/// Implementors must provide the implementation of `def_id` which will be used to retrieve
+/// information about its definition.
```

Comment:
```suggestion
/// information about the crate's definition.
```
we can make this clearer